<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATCDialog</class>
 <widget class="QDialog" name="ATCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: #000000;
}

QFrame#frameDialog
{
	background-color: #000000;
	border-style: outset;
	border-width: 1px;
	border-color: #3e3e3e;
}

QPushButton#buttonClose
{
	color: #828282;
	background-color: #000000;
	border-style: none;
	font: bold;
}

QPushButton#buttonClose::hover
{
	color: #ffffff;
	background-color: #000000;
	border-style: none;
	font: bold;
}

QPushButton#buttonMinMax
{
	color: #828282;
	background-color: #000000;
	border-style: none;
	font: bold;
}

QPushButton#buttonMinMax::hover
{
	color: #ffffff;
	background-color: #000000;
	border-style: none;
	font: bold;
}

QLabel#labelTitle
{
	color: #828282;
	font: bold;
}
</string>
  </property>
  <widget class="QFrame" name="frameDialog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frameTitleBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonMinMax">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>20</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>â–²</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonClose">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>20</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
